.env

```
@@ -0,0 +1,19 @@
+# 브라우저에서도 쓰이는 값
+NEXT_PUBLIC_GTM_ID=GTM-TN5ZTHP9
+NEXT_PUBLIC_GA_ID=G-YYZYZ94M1P
```

dotenv-linter (3.3.0)

[warning] [UnorderedKey] The NEXT_PUBLIC_GA_ID key should go before the NEXT_PUBLIC_GTM_ID key

(UnorderedKey)